.babelrc

```diff
@@ -1,4 +1,14 @@
 {
     "presets": ["env", "react"],
-    "plugins": ["transform-object-rest-spread", "styled-jsx/babel"]
-}
\ No newline at end of file
+    "env": {
+        "production": {
+            "plugins": ["transform-object-rest-spread", "styled-jsx/babel"]
+        },
+        "development": {
+            "plugins": ["transform-object-rest-spread", "styled-jsx/babel"]
+        },
+        "test": {
+            "plugins": ["transform-object-rest-spread", "styled-jsx/babel-test"]
```

Comment:
Here we use (a brand new) plugin `babel-test`, so that Jest will use Babel and styled-jsx, but won't parse the classes to `jsx-1324` or whatever, so that we can make assertions on them!